plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidRemoteIdentifierSupplier.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.druid;
+
+import io.trino.plugin.jdbc.mapping.RemoteIdentifierSupplier;
+
+import java.sql.Connection;
+
+public class DruidRemoteIdentifierSupplier
+        implements RemoteIdentifierSupplier
+{
+    @Override
+    public String toRemoteIdentifier(Connection connection, String identifier)
+    {
```

Comment:
Should we add some comment why we return plain identifier?

Reply:
indeed, also this can be a lambda now so comment can live in `DruidJdbcClientModule`.

Reply:
addressed